### ISSUE TITLE
Build broken under clang

### ISSUE THREAD

**Issue by danluu** [BitFunnel/BitFunnel#214]
~~~
lTest.dir/BitFunnelToolTest.cpp.o.d -o tools/BitFunnel/test/CMakeFiles/BitFunnelToolTest.dir/BitFunnelToolTest.cpp.o -c ../tools/BitFunnel/test/BitFunnelToolTest.cpp
../tools/BitFunnel/test/BitFunnelToolTest.cpp:83:17: error: ISO C++11 does not allow conversion from string literal to 'char *const' [-Werror,-Wwritable-strings]
                "BitFunnel",
                ^
../tools/BitFunnel/test/BitFunnelToolTest.cpp:84:17: error: ISO C++11 does not allow conversion from string literal to 'char *const' [-Werror,-Wwritable-strings]
                "statistics",
                ^
../tools/BitFunnel/test/BitFunnelToolTest.cpp:85:17: error: ISO C++11 does not allow conversion from string literal to 'char *const' [-Werror,-Wwritable-strings]
                "manifest.txt",
                ^
../tools/BitFunnel/test/BitFunnelToolTest.cpp:86:17: error: ISO C++11 does not allow conversion from string literal to 'char *const' [-Werror,-Wwritable-strings]
                "config"
                ^
../tools/BitFunnel/test/BitFunnelToolTest.cpp:101:17: error: ISO C++11 does not allow conversion from string literal to 'char *const' [-Werror,-Wwritable-strings]
                "BitFunnel",
                ^
../tools/BitFunnel/test/BitFunnelToolTest.cpp:102:17: error: ISO C++11 does not allow conversion from string literal to 'char *const' [-Werror,-Wwritable-strings]
                "termtable",
                ^
../tools/BitFunnel/test/BitFunnelToolTest.cpp:103:17: error: ISO C++11 does not allow conversion from string literal to 'char *const' [-Werror,-Wwritable-strings]
                "config"
                ^
../tools/BitFunnel/test/BitFunnelToolTest.cpp:118:17: error: ISO C++11 does not allow conversion from string literal to 'char *const' [-Werror,-Wwritable-strings]
                "BitFunnel",
                ^
../tools/BitFunnel/test/BitFunnelToolTest.cpp:119:17: error: ISO C++11 does not allow conversion from string literal to 'char *const' [-Werror,-Wwritable-strings]
                "repl",
                ^
../tools/BitFunnel/test/BitFunnelToolTest.cpp:120:17: error: ISO C++11 does not allow conversion from string literal to 'char *const' [-Werror,-Wwritable-strings]
                "config"
                ^
~~~

I took a quick look at fixing this one way and it was an annoying change. I can take a look at better fix tomorrow.

**Comment by MikeHopcroft** [comment]
I can fix it. Still have to figure out why I'm not getting the email. I checked the badge on GitHub and it seemed to be passing.

Sent from my Windows Phone
________________________________

**Comment by danluu** [comment]
That's because we don't have clang CI right now.

I wonder if we shouldn't just try to set up LLVM on appveyor since appveyor has been great for us with VC++, and they seem to keep their llvm version up to date.

**Comment by danluu** [comment]
Seems to be fixed by 8f49fe39328030748186df75d744296d40019a42
